libraries/AP_HAL_ChibiOS/hwdef/CubeOrangePlus-dv/defaults.parm

```diff
@@ -1,630 +1,467 @@
-SR2_EXT_STAT 2
-SR2_PARAMS 0
-SR2_POSITION 10
-SR2_RAW_CTRL 0
-SR2_RAW_SENS 2
-SR2_RC_CHAN 2
-SR3_ADSB 0
-SR3_EXTRA1 0
-SR3_EXTRA2 0
-SR3_EXTRA3 0
-SR3_EXT_STAT 0
-SR3_PARAMS 0
-SR3_POSITION 0
-SR3_RAW_CTRL 0
-SR3_RAW_SENS 0
-SR3_RC_CHAN 0
-SR4_ADSB 0
-SR4_EXTRA1 0
-SR4_EXTRA2 0
-SR4_EXTRA3 0
-SR4_EXT_STAT 0
-SR4_PARAMS 0
-SR4_POSITION 0
-SR4_RAW_CTRL 0
-SR4_RAW_SENS 0
-SR4_RC_CHAN 0
-SRTL_ACCURACY 2
-SRTL_OPTIONS 0
-SRTL_POINTS 500
-
-SUPER_SIMPLE 0
-SURFTRAK_MODE 0
-SURFTRAK_TC 1
-TCAL_ENABLED 0
-TELEM_DELAY 0
-TERRAIN_OFS_MAX 0
-THR_DZ 35
-TKOFF_RPM_MAX 0
-TKOFF_RPM_MIN 1200
-TKOFF_SLEW_TIME 2
-TKOFF_THR_MAX 0.9
-WPNAV_ACCEL 350
-WPNAV_ACCEL_C 250
-WPNAV_ACCEL_Z 150
-WPNAV_JERK 5
-WPNAV_RADIUS 100
-WPNAV_RFND_USE 0
-WPNAV_SPEED 600
-WPNAV_SPEED_DN 300
-WPNAV_SPEED_UP 500
-WPNAV_TER_MARGIN 10
-WP_NAVALT_MIN 0
-WP_YAW_BEHAVIOR 2
-WVANE_ENABLE 0
+
+AHRS_EKF_TYPE,3
+AHRS_GPS_USE,2
+AHRS_ORIENTATION,2
+AHRS_TRIM_X,0
+AHRS_TRIM_Y,0
+AHRS_TRIM_Z,0
+
+ANGLE_MAX,2500
+
+ARMING_CHECK,1
+ARMING_MAGTHRESH,200
+
+ATC_ACCEL_P_MAX,50000
+ATC_ACCEL_R_MAX,50000
+ATC_ACCEL_Y_MAX,13000
+ATC_ANG_LIM_TC,2
+ATC_ANG_PIT_P,12
+ATC_ANG_RLL_P,11.5
+ATC_ANG_YAW_P,6.5
+ATC_ANGLE_BOOST,1
+ATC_INPUT_TC,0.2
+ATC_RAT_PIT_D,0.0045
+ATC_RAT_PIT_FLTD,10
+ATC_RAT_PIT_FLTT,10
+ATC_RAT_PIT_I,0.095
+ATC_RAT_PIT_IMAX,0.5
+ATC_RAT_PIT_P,0.095
+ATC_RAT_RLL_D,0.0038
+ATC_RAT_RLL_FLTD,10
+ATC_RAT_RLL_FLTT,10
+ATC_RAT_RLL_I,0.1
+ATC_RAT_RLL_IMAX,0.5
+ATC_RAT_RLL_P,0.1
+ATC_RAT_YAW_FLTD,10
+ATC_RAT_YAW_FLTE,2.2
+ATC_RAT_YAW_FLTT,10
+ATC_RAT_YAW_I,0.07
+ATC_RAT_YAW_IMAX,0.5
+ATC_RAT_YAW_P,0.7
+ATC_RATE_FF_ENAB,1
+ATC_RATE_P_MAX,180
+ATC_RATE_R_MAX,180
+ATC_RATE_Y_MAX,180
+ATC_SLEW_YAW,6000
+ATC_THR_G_BOOST,0
+ATC_THR_MIX_MAN,0.1
+ATC_THR_MIX_MAX,0.7
+ATC_THR_MIX_MIN,0.1
+
+AVOID_ACCEL_MAX,3
+AVOID_ALT_MIN,0
+AVOID_ANGLE_MAX,1000
+AVOID_BACKUP_DZ,0.1
+AVOID_BACKUP_SPD,0.75
+AVOID_BEHAVE,0
+AVOID_DIST_MAX,5
+AVOID_ENABLE,3
+AVOID_MARGIN,2
+
+BATT_AMP_PERVLT,50
+BATT_ARM_VOLT,42
+BATT_CAPACITY,22000
+BATT_CRT_VOLT,38
+BATT_FS_CRT_ACT,1
+BATT_FS_LOW_ACT,2
+BATT_FS_VOLTSRC,0
+BATT_LOW_TIMER,10
+BATT_LOW_VOLT,40
+BATT_MONITOR,4
+BATT_VOLT_MULT,15.31
+
+BRD_BOOT_DELAY,1000
+BRD_HEAT_LOWMGN,20
+BRD_HEAT_TARG,45
+BRD_SAFETY_DEFLT,0
+BRD_TYPE,3
+BRD_VBUS_MIN,4.3
+
+CAN_D1_PROTOCOL,1
+CAN_D1_PROTOCOL2,0
+CAN_D1_UC_NODE,10
+CAN_D1_UC_NTF_RT,5
+CAN_D1_UC_OPTION,2
+CAN_D1_UC_POOL,16384
+CAN_D1_UC_SRV_RT,1
+CAN_D2_PROTOCOL,1
+CAN_D2_PROTOCOL2,0
+CAN_D2_UC_NODE,10
+CAN_D2_UC_NTF_RT,20
+CAN_D2_UC_POOL,16384
+CAN_D2_UC_SRV_RT,50
+CAN_P1_BITRATE,1000000
+CAN_P1_DRIVER,1
+CAN_P1_FDBITRATE,8
+CAN_P2_BITRATE,1000000
+CAN_P2_DRIVER,2
+CAN_P2_FDBITRATE,8
+CAN_SLCAN_CPORT,1
+CAN_SLCAN_SDELAY,1
+CAN_SLCAN_SERNUM,-1
+CAN_SLCAN_TIMOUT,2
+
+COMPASS_AUTO_ROT,2
+COMPASS_AUTODEC,1
+COMPASS_CAL_FIT,16
+COMPASS_DISBLMSK,32
+COMPASS_ENABLE,1
+COMPASS_FLTR_RNG,0
+COMPASS_LEARN,0
+COMPASS_MOTCT,0
+COMPASS_USE,1
+COMPASS_USE2,0
+COMPASS_USE3,0
+
+DID_BARO_ACC,0
+DID_CANDRIVER,0
+DID_ENABLE,1
+DID_MAVPORT,1
+DID_OPTIONS,0

```

Comment:
Just as a comment, this will remove the ODID prearm check so the drone can be armed even if ODID does not work.

Reply:
I assume you mean DID_OPTION = 0 ?... I know and you can't arm if you have "operator location lost" message (which is almost continuously with the Herelink), but It was set like this on our Golden KDV1-0001 KOBRA. Not programmed or set like that by me. Anyway if you set it to 1 you will get a lot of ODID error messages continuously. I mentioned it to Vincent last week.

Reply:
So to update on info from Daan; 
DID_OPTIONS = 1.. will allow me to arm, but I get “PreArm: OpenDroneID: SYSTEM not available” warning messages. I put it back to DID_OPTIONS = 0

PreArm: OpenDroneID: SYSTEM not available translates to "OpenDroneID misconfiguration", https://ardupilot.org/copter/docs/common-prearm-safety-checks.html.

For the purpose of reaching a stable parameterlist for the demo drones, I argue to let DID_OPTIONS=0 for now, but this should be revisited because it is on contradiction with "EN4709-001 Product Requirements and Verification" (the C3 requirements).

Reply:
Just a bit more info.. Last week Sept 19th I was able to arm despite the warnings like described in Vincent's comment above, but on my first flight with KDV1-0004 on August 2nd I wasn't able to arm until I changed DID_OPTIONS to 0.

Reply:
We need a decision to be made on this. 0 - will simplify things a lot in development but will hit us in the longer run.
@vtofterup ?

Reply:
I do not like it, but, in the scope of getting DEMO drones up and flying quickly we have to leave at 0 for now. 

I hope that it will not hit us that hard afterwards - But I agree with you, I just want the DEMO drones up and flying.

Reply:
Ok.